Documentation/Setup/Page/Index.rst

```diff
@@ -230,48 +230,76 @@ cssInline
 
 .. _setup-page-footerdata:
 
-footerData
-==========
+footerData.[array]
+==================
 
 .. container:: table-row
 
    Property
-         footerData
+         footerData.[array]
 
    Data type
-         ->CARRAY
+         :ref:`cObject <data-type-cobject>`
 
    Description
          Same as :ref:`setup-page-headerData` above,
          except that this block gets included at the bottom of the page
          (just before the closing :html:`</body>` tag).
 
+         The :ts:`footerData` property takes any number of numeric keys, each representing a one
+         cObject.
```

Comment:
Would slightly rephrase. Suggestion:

> The :ts:`footerData` property contains any number of numeric keys, each representing one cObject.

And I would add that exact phrase to any of the similar properties (headerData, jsFooterInline, cssInline)